src/nginx/varify/varify.go

```diff
@@ -15,6 +18,11 @@ func main() {
 	filename := os.Args[1]
 	localModulePath := os.Args[2]
 	globalModulePath := os.Args[3]
+	resolvConfPath := "/etc/resolv.conf"
+	if len(os.Args) > 4 {
+		resolvConfPath = os.Args[4]
+	}
+	defaultNameserver := "169.254.0.2" // https://github.com/cloudfoundry/bosh-dns-release/blob/master/jobs/bosh-dns/spec#L36-L38
```

Comment:
What if somebody _doesn't_ configure this as their default nameServer? Shouldn't we read this value from the default resolveConfPath?

Reply:
`defaultNameserver` comes from this doc: https://github.com/cloudfoundry/bosh-dns-release/blob/master/jobs/bosh-dns/spec#L36-L38 - the chance that it will change is close to zero.
I use this value only when there is problem with reading "/etc/resolv.conf".
Is better to return the empty value, default or maybe raise fatal error?